Makefile

```diff
@@ -9,18 +9,18 @@ IMAGENAME ?= resource-topology-exporter
 IMAGETAG ?= latest
 RTE_CONTAINER_IMAGE ?= quay.io/$(REPOOWNER)/$(IMAGENAME):$(IMAGETAG)
 
-LDFLAGS = -ldflags "-s -w"
-VERSION := $(shell git describe --tags)
-ifneq ($(VERSION),)
-LDFLAGS = -ldflags "-s -w -X github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/version.version=$(VERSION)"
-endif
 
 .PHONY: all
 all: build
 
+.PHONY: build-tools
+build-tools: outdir _out/git-semver
+
 .PHONY: build
-build: outdir
-	$(COMMONENVVAR) $(BUILDENVVAR) go build $(LDFLAGS) -o _out/resource-topology-exporter cmd/resource-topology-exporter/main.go
+build: build-tools
+	$(COMMONENVVAR) $(BUILDENVVAR) go build \
+	-ldflags "-s -w -X github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/version.version=v$(shell _out/git-semver)" \
```

Comment:
Shouldn't this 'v' be there?  starting with a prefix this would not be a semver compliant version.

Reply:
didn't check the spec: this is as close as possible to what the old code did. So let me remove the "v"